src/webview/create-component/createComponentLoader.ts

```diff
@@ -566,6 +583,32 @@ export default class CreateComponentLoader {
             });
         }
     }
+
+    static getInitialWorkspaceFolder(): string | undefined {
+        const wsFolders: path.ParsedPath[] = [];
+        vscode.workspace.rootPath && wsFolders.push(path.parse(vscode.workspace.rootPath));
+        vscode.workspace?.workspaceFolders?.forEach((f) => wsFolders.push(path.parse(f.uri.fsPath)));
+
+        if (wsFolders.length >= 0) {
```

Comment:
This condition is always true. I think you mean `wsFolder.length > 0`, since you access the first element of the array below.

Reply:
Yes, you're right.

Reply:
Fixed